frontend/public/components/monitoring/alerting.tsx

```diff
@@ -4,7 +4,8 @@ import { Button, Popover } from '@patternfly/react-core';
 import { sortable } from '@patternfly/react-table';
 import * as React from 'react';
 import { Helmet } from 'react-helmet';
-import { useTranslation } from 'react-i18next';
+import { Trans, useTranslation } from 'react-i18next';
```

Comment:
I believe this is duplicating #7386, cc @kyoto

Reply:
Update the file according to #7386.